archive/app/controllers/ArchiveController.scala

```diff
@@ -64,7 +64,7 @@ class ArchiveController(dynamoDB: DynamoDB) extends Controller with Logging with
         Some(path)
       case _ => None
     }
-    s"http://${normalised.getOrElse(path)}"
+    s"https://${normalised.getOrElse(path)}"
```

Comment:
Indeed. You are right, it might require some refactoring. For now I give you a 👍 💃